src/libraries/System.Security.Cryptography.Encoding/src/System.Security.Cryptography.Encoding.csproj

```diff
@@ -5,6 +5,9 @@
     <TargetFrameworks>$(NetCoreAppCurrent)-Windows_NT;$(NetCoreAppCurrent)-Unix;$(NetCoreAppCurrent)-OSX;$(NetCoreAppCurrent)-iOS;$(NetCoreAppCurrent)-tvOS</TargetFrameworks>
     <Nullable>enable</Nullable>
   </PropertyGroup>
+  <PropertyGroup>
+    <GeneratePlatformNotSupportedAssemblyMessage Condition="'$(TargetsBrowser)' == 'true'">SR.Cryptography_PlatformNotSupported_Browser</GeneratePlatformNotSupportedAssemblyMessage>
```

Comment:
The two OS functions that this library uses are:

* Tell me what the friendly name of this OID is (Windows, Linux)
* Pretty print this identified item (Windows, Linux)

macOS has no OS fallbacks, it just uses managed implementations with limited string support; so Browser could just build with the "OSX" flavor.

That said, there's not a whole lot I'd expect anyone to do with this assembly without the rest of crypto, so maybe PNSE is best for now.

Reply:
I think PNSE is better for now.